indica/readers/adas.py

```diff
@@ -133,7 +131,14 @@ def get_adf11(self, quantity: str, element: str, year: str) -> DataArray:
                     date = new_date
                 data[i, ...] = np.fromfile(f, float, nd * nt, " ").reshape((nt, nd))
         gen_type = ADF11_GENERAL_DATATYPES[quantity]
-        spec_type = ORDERED_ELEMENTS[z]
+        spec_type_tmp = [value[2] for value in ELEMENTS.values() if value[0] == z]
+        # This is basically only needed for hydrogen, deuterium and tritium,
+        # where z=1 for all of them. Other elements in the datatypes.py have a
+        # unique atomic number.
```

Comment:
Could do this find/assert the opposite way to simplify things: find the element based off `spec_type` since that's the unique part and then assert the Z from that element is equal to `z` from the file

Reply:
Yep definitely, I'll make the change